src/main/kotlin/com/google/androidstudiopoet/models/ProjectBlueprint.kt

```diff
@@ -79,6 +80,7 @@ class ProjectBlueprint(private val projectConfig: ProjectConfig) {
     fun saveDependencies() {
         val graphFileName = projectRoot.joinPath("dependencies.dot")
         File(graphFileName).printWriter().use { out -> out.print(dependenciesGraphString())}
+        DependencyImageGenerator().generate(this)
```

Comment:
Please move the whole `saveDependencies()` method to `SourceModuleGenerator`, then inject the `DependencyImageGenerator()` into it, and use `FileWriter` to manipulate with files. And please rename `graphFileName` to `dependenciesGraphFileName` and keep it as a field in this class.
I'm sorry I missed you previous PR.